script.js

```diff
@@ -1,11 +1,37 @@
 $(document).ready(function() {
-  listSpots();
+  var waypointsArray = [];
+
   function listSpots() {
-    $.getJSON("topspots.json", function(data) { // pulling in the data
+    $.getJSON("topspots.json", function(data) {
       $.each(data, function(index, item) {
+
+        // concat the table cell HTML and variables
         var currentRow = "<tr><td>" + item.name + "</td><td>" + item.description + "</td><td><a href='https://www.google.com/maps?q=" + item.location + "'><button id='submitButton' class='btn btn - primary'>Open in Maps</button></a></td></tr>";
         $(currentRow).appendTo('#goHere');
```

Comment:
Consider renaming the location where you are putting data to be more meaningful for developers to discover.